packages/react-i18n/package.json

```diff
@@ -35,12 +35,14 @@
   },
   "dependencies": {
     "@shopify/i18n": "^0.1.5",
+    "@shopify/javascript-utilities": "^2.4.0",
```

Comment:
Weren't these actually missing dependencies of `i18n`? Also, can we just remove these in favour of our own stuff?

Reply:
it is actually in `react-i18n`,  not `i18n`. The error message is as follow.
> Unable to resolve module `@shopify/javascript-utilities/decorators` from `/Users/path/to/node_modules/@shopify/react-i18n/dist/i18n.js: Module `@shopify/javascript-utilities/decorators` does not exist in the Haste module map`

but you are right,  I can switch this one out to use our own decorator. 
thou our version of memoize skip server execution, do you think that would be a problem here?
(if not I will switch both the decorator & memoizeFn)